symfony/migrations/Version20210413150150.php

```diff
@@ -0,0 +1,467 @@
+<?php
```

Comment:
Dit moet niet mee in migrations, wordt voorzien op initiële build

Reply:
Enkel icapps_ prefixed Entities dan?